lms/djangoapps/support/static/support/jsx/program_enrollments/inspector.jsx

```diff
@@ -2,55 +2,174 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Button, InputText, StatusAlert, InputSelect } from '@edx/paragon';
 
-export const ProgramEnrollmentsInspectorPage = props => (
-  <form method="get">
-    {props.errors.map(errorItem => (
-      <StatusAlert
-        open
-        dismissible={false}
-        alertType="danger"
-        dialog={errorItem}
-      />
-    ))}
-    <div key="edX_accounts">
-      <InputText
-        name="edx_user"
-        label="edX account username or email"
-        value={(props.learnerInfo && props.learnerInfo.user && props.learnerInfo.user.username) || ''}
-      />
-    </div>
-    <div key="school_accounts">
-      <InputSelect
-        name="IdPSelect"
-        label="Learner Account Providers"
-        value="Select One"
-        options={
-          props.orgKeys
-        }
-      />
-
-      <InputText
-        name="external_user_key"
-        label="Institution user key from school. For example, GTPersonDirectoryId for GT students"
-        value={(props.learnerInfo && props.learnerInfo.user && props.learnerInfo.user.external_user_key) || ''}
-      />
+
+const renderUserSection = userObj => (
+  <div>
+    <h3>
+      edX account Info
+    </h3>
+    <div><span>Username: </span> {userObj.username}</div>
+    <div><span>Email: </span> {userObj.email}</div>
+    {userObj.external_user_key && (
+      <div><span>External User Key</span>: {userObj.external_user_key}</div>
+    )}
+    {userObj.sso ? (
+      <div>
+        <h4>Single Sign On Record:</h4>
+        <div><span>UID: </span>{userObj.sso.uid}</div>
+        <div><span>Identity Provider: </span>{userObj.sso.provider}</div>
+      </div>
+    ) : (
+      <div> There is no Single Signed On record associated with this user!</div>)}
+    <hr />
+  </div>
+);
+
+
+const renderVerificationSection = verificationStatus => (
+  <div>
+    <h3>
+      ID Verification
+    </h3>
+    <div><span>Status: </span> {verificationStatus.status}</div>
+    <div><span>Verification Error: </span> {verificationStatus.error}</div>
+    <div><span>Should Display Verification: </span> {verificationStatus.should_display}</div>
```

Comment:
What does "Should Display Verification" mean? I poked around and it seems like it means "should_display only refers to displaying the verification attempt status to a user". Is that useful information for the support team?

Reply:
Good point. I can remove this piece of information for now. If you found it useful, I'll add later in a different PR.